test/csuite/wt5212_incremental_backup/main.c

```diff
@@ -0,0 +1,659 @@
+      buf, sizeof(buf), "%s -R -h %s.%d list | grep %s", WT_UTIL, home_incr, counter, uri));
+    ret = system(buf);
+    testutil_assertfmt(
+      ret != 0, "%s dropped, but table exists in %s.%d\n", uri, home_incr, counter);
+
+    /*
+     * Clean up the folder.
+     */
+    testutil_check(__wt_snprintf(buf, sizeof(buf), "rm -rf %s.%d", home_incr, counter));
+    testutil_check(system(buf));
+
+    new_table = false;
+}
+
+/*
+ * This function will create previously dropped table uri (table:main) and add different content to
+ * it, take backups and validate the backups.
+ */
+static void
+create_dropped_table_add_new_content(WT_SESSION *session)
+{
+    testutil_check(session->create(session, uri, "key_format=i,value_format=S"));
+    add_work(session, uri, false);
+    take_full_backup(session);
+    take_incr_backup(session);
+    testutil_check(compare_backups(uri));
+}
+
+/*
+ * This function will insert bulk data in logged and not-logged table, take backups and validate the
+ * backups.
+ */
+static void
+insert_bulk_data(WT_SESSION *session)
+{
+    /*
+     * Set this variable to true so that function take_incr_backup copies full file instead of
+     * writing blocks to the new created objects
+     */
+    new_table = true;
+    /*
+     * Insert bulk data into logged table.
+     */
+    testutil_check(session->create(session, uri3, "key_format=i,value_format=S"));
+    add_work(session, uri3, true);
+    take_full_backup(session);
+    take_incr_backup(session);
+    testutil_check(compare_backups(uri3));
+
+    /*
+     * Insert bulk data into not logged table.
+     */
+    testutil_check(
+      session->create(session, uri4, "key_format=i,value_format=S,log=(enabled=false)"));
+    add_work(session, uri4, true);
+    take_full_backup(session);
+    take_incr_backup(session);
+    testutil_check(compare_backups(uri4));
+    new_table = false;
+}
+
+int
+main(int argc, char *argv[])
+{
+    WT_CONNECTION *wt_conn;
+    WT_SESSION *session;
+    int i;
+    char cmd_buf[256];
+
+    (void)argc; /* Unused variable */
+    (void)testutil_set_progname(argv);
+
+    testutil_check(
+      __wt_snprintf(cmd_buf, sizeof(cmd_buf), "rm -rf %s && mkdir -p %s/%s", home, home, logpath));
+    testutil_check(system(cmd_buf));
+    testutil_check(wiredtiger_open(home, NULL, CONN_CONFIG, &wt_conn));
+
+    setup_directories();
+    testutil_check(wt_conn->open_session(wt_conn, NULL, NULL, &session));
+    testutil_check(session->create(session, uri, "key_format=i,value_format=S"));
+    testutil_check(session->create(session, uri2, "key_format=i,value_format=S"));
+
+    printf("*** Add data, checkpoint, take backups and validate ***\n");
+    add_data_validate_backups(session);
+
+    printf("*** Remove old records and validate ***\n");
+    remove_all_records_validate(session);
+
+    /*
+     * Close and re-open the connection to drop existing table.
+     */
+    testutil_check(wt_conn->close(wt_conn, NULL));
```

Comment:
You still should not need to do this. I think the EBUSY error is likely to be a cursor left open on the table you want to drop. Check for a cursor being left open on this table somewhere. It is not harmful exactly, but should not be necessary.